src/components/Blog.jsx

```diff
@@ -0,0 +1,31 @@
+function Blog({ blogEntries }) {
```

Comment:
I think I remember now why this was a React component for sessions and this shouldn't be one for blog:
The session page does this thing where it shows the next session based on the user's local time.
And that needs to happen on the client, while here I believe this can all be rendered on the server.
I'll fix this one day, we can merge this in as is for now.

Reply:
Thanks!